spec/derivative_services/image_derivative_service_spec.rb

```diff
@@ -41,17 +41,19 @@
     end
   end
 
-  it "creates a thumbnail and attaches it to the fileset" do
-    derivative_service.new(valid_change_set).create_derivatives
-
-    reloaded = query_service.find_by(id: valid_resource.id)
-    members = query_service.find_members(resource: reloaded)
-    derivative = members.find { |x| x.use.include?(Valkyrie::Vocab::PCDMUse.ServiceFile) }
-
-    expect(derivative).to be_present
-    derivative_file = Valkyrie::StorageAdapter.find_by(id: derivative.file_identifiers.first)
-    image = MiniMagick::Image.open(derivative_file.io.path)
-    expect(image.width).to eq 105
-    expect(image.height).to eq 150
+  describe '#create_derivatives' do
```

Comment:
Does this change something, or just a refactor to the test? If the latter, maybe a different PR? I think there's a larger discussion to be had about how we want to format our tests before we start re-building them.

Reply:
Yes, you are correct, this was just a refactoring.  I was initially aiming to extend this in order to test against at least one additional `context` in which an adapter which leverages the `Tempfile` is used.